articles/server-platforms/symfony/01-login.md

```diff
@@ -11,6 +11,14 @@ This tutorial and seed project have been tested with the following:
 * Symfony 2.8
 :::
 
+<%= include('../_includes/_package', {
```

Comment:
@mehreencs87 This `../_includes/_package` does not exist, the valid one is `../../_includes/_package`. Please update this.

Also, you should add one more line here, according to the latest [contributing guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects), the `githubUrl` option. Its value should be the full URL to the sample, i.e. `https://github.com/auth0-samples/auth0-symfony-php-web-app/tree/master/00-Starter-Seed`